src/opa.js

```diff
@@ -49,13 +48,58 @@ function _dumpJSON(wasmInstance, memory, addr) {
     return JSON.parse(s);
 }
 
-
-function builtinPlus(a, b) {
-    return a+b;
-}
-
 const builtinFuncs = {
-    plus: builtinPlus,
+    // Numbers
+    plus: (a, b) =>  a + b,
```

Comment:
Just thinking outloud, this might be a good opportunity to consider the file/directory structure for the SDK. It might make sense to move these out into a separate directory and then sub files for the groups. Something like:
```
./src/builtins
./src/builtins/numbers.js
./src/builtins/aggregates.js
./src/builtins/arrays.js
.
.
.
```

The reasoning being that over time these are likely to not be so simple, and would likely benefit from being more standalone functions with room for docs and stuff.

If we do this we'd probably want to have some sort of registration step (similar to how they are done in OPA itself where each file of builtins handles registering themselves with the builtin name).

Thoughts?

Reply:
Probably begin by breaking up the functions without registration of a builtin for now, I'll import into opa from the builtins.